### PR TITLE
Various fixes

### PR DESCRIPTION
**Pull request by niol**
Various fixes detailed in the individual patches.

### CODE REVIEW

**Comment by lumiru** [comment]
I think you at least need to add a IfModule on .htaccess (or modify install/upgrade documentation about Apache configuration) :

```apache
<IfModule mod_authz_core.c>
                        Require all granted
</IfModule>
```

File: public/lang/fr.ini [hunk 1]
```diff
@@ -49,3 +49,4 @@ lang_login_username=Identifiant
 lang_login_password=Mot de passe
 lang_no_title=Sans titre
 lang_error=Une erreur est survenue
+lang_streamerror=Erreur lors du chargement de la liste, essayer à nouveau?
```
Comment (bendem) [inline public/lang/fr.ini]: Missing a space before `?`

**Comment by niol** [comment]
Regarding the ```<IfModule mod_authz_core.c>```, as a selfoss user, I'd prefer a ```500 Internal Server Error``` rather than my password hash being exposed and the rule not being applied silently.

**Comment by lumiru** [comment]
You make a point.

So you should add something about the minimum version in installation instructions (or as known issue).

File: daos/pgsql/Database.php [hunk 2]
```diff
@@ -186,19 +184,18 @@ public function __construct() {
...
+                        'INSERT INTO version (version) VALUES (8);'
+                    ));
+                }
+                if(strnatcmp($version, "9") < 0) {
+                    \F3::get('db')->exec(array(
+                        'ALTER TABLE items ADD shared BOOLEAN;',
+                        'INSERT INTO version (version) VALUES (9);'
+                    ));
+                }
                 }
```
Comment (jtojnar) [inline daos/pgsql/Database.php]: What is this double closing bracket?

**Comment by SSilence** [comment]
@niol may you send me a short email (tobias.zeising@aditu.de) that I have your email address?

**Pull request merged**